app/main/views/audits.py

```diff
@@ -37,6 +39,7 @@
     "frameworks": models.Framework.slug,
     "users": models.User.id,
     "briefs": models.Brief.id,
+    "outcomes": models.Outcome.external_id,
```

Comment:
need to add `brief-responses` here

Reply:
Do I? Didn't think `BriefResponse`s had a special external id.

Reply:
Jeez, looks like I need to include it anyway...